rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date
+
+  def initiallize(date, book, _peson)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] The constructor of Rental should be modified so Book and Person are set in it to break the many-to-many relationship. Link to the example article: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship

Reply:
done